File: exs/mundo_1/ruby/017.rb
=begin
Desafio 017

Problema: Faça um programa que leia o comprimento do cateto oposto e
          do cateto adjacente de um triângulo retângulo. Calcule e
          mostre o comprimento da hipotenusa.

Resolução do problema:
=end

print"Digite o valor do cateto oposto: "
co = gets.chomp.to_f

print"Digite o valor do cateto adjacente: "
ca = gets.chomp.to_f

h = ((co**2)+(ca**2))**0.5

puts"A hipotenusa vale #{h}."

File: exs/mundo_1/golang/017.go
/*
Desafio 017

Problema: Faça um programa que leia o comprimento do cateto oposto e
          do cateto adjacente de um triângulo retângulo. Calcule e
          mostre o comprimento da hipotenusa.

Resolução do problema:
*/

package main

import (
	"fmt"
	"math"
)

var catOP, catAD float64

func main(){
	fmt.Print("Digite o valor do cateto adjacente: ")
	fmt.Scanln(&catAD)
	fmt.Print("Digite o valor do cateto oposto: ")
	fmt.Scanln(&catOP)

	hipotenusa := math.Hypot(catAD, catOP)

	fmt.Println("O valor da hipotenusa é igual a", hipotenusa)

}
